File: cash.c
#include<cs50.h>
#include<stdio.h>

int main(void)
{

    int n = -1;
    while(n < 0)
    {
        n = get_int("Changed owed: ");
    }
    int res = 0;
    while(n > 0)
    {
        if (n >= 25)
        {
            res++;
            n -= 25;
        }
        else if (n >= 10)
        {
            res++;
            n -= 10;
        }
        else if (n >= 5)
        {
            res++;
            n -= 5;
        }
        else
        {
            res++;
            n--;
        }
    }
    printf("%i\n", res);
}

File: cash.py
import cs50
while (True):
    try:
        cash = cs50.get_float("Change: ")
        if cash > 0:
            break
    except ValueError:
        pass
cash = round(cash * 100)
res = 0
while (cash >= 25):
    cash -= 25
    res += 1
while (cash >= 10):
    cash -= 10
    res += 1
while (cash >= 5):
    cash -= 5
    res += 1
while (cash >= 1):
    cash -= 1
    res += 1
print(res)
